templates/firefox-ios.xliff

```diff
@@ -1414,10 +1490,22 @@
         <source>Save pages to your Reading List by tapping the book plus icon in the Reader View controls.</source>
         <note>See http://mzl.la/1LXbDOL</note>
       </trans-unit>
+      <trans-unit id="ScanQRCode.Error.OK.Button">
+        <source>OK</source>
+        <note>OK button to dismiss the error prompt.</note>
+      </trans-unit>
       <trans-unit id="ScanQRCode.Instructions.Label">
         <source>Align QR code within frame to scan</source>
         <note>Text for the instructions label, displayed in the QR scanner view</note>
       </trans-unit>
+      <trans-unit id="ScanQRCode.InvalidDataError.Message">
+        <source>The data is invalid</source>
+        <note>Text of the prompt user the data is invalid</note>
```

Comment:
I assume this should be "Text of the prompt user data is invalid" or something like that?